stdlib/3/os/__init__.pyi

```diff
@@ -343,10 +344,16 @@ if sys.version_info >= (3, 3):
     def replace(src: _PathType, dst: _PathType) -> None: ...
 def rmdir(path: _PathType) -> None: ...
 if sys.version_info >= (3, 6):
+    class ScandirIterator(Iterable):
+        def __iter__(self) -> Iterator[DirEntry[str]]: ...
+        def __next__(self) -> DirEntry[str]: ...
+        def __enter__(self: _T) -> _T: ...
+        def __exit__(self, exc_type: Optional[type], exc_val: Optional[Exception],
```

Comment:
exc_type should be `Optional[Type[BaseException]]` and `exc_val` should be `Optional[BaseException]`.